docker-compose.local.yaml

```diff
@@ -1,21 +1,23 @@
 version: '3.4'
+
+# This override file should be used in development environments to
+# connect this service to sibling services from other repositories
+
 services:
   mine-support-calculation-service:
-    command:  ./wait-for amq:8161 -- npm run start-watch
+    command: >
+      sh -c "
+        scripts/wait-for/wait-for mine-support-message-queue:5672 --

```

Comment:
This script fails to run as service name in docker-compose is "mine-support-*calculation*-message-queue".

Reply:
Not quite. You need to use `docker-compose up` to run this service with its own message queue (Docker Compose will use `docker-compose.yaml` and `docker-compose.override.local` by default).

`script/start` is now intended to be used when there's already a local message queue instance running (e.g. via the development repo which has an open PR for this).

Reply:
Ah ok, I see.   So if we want to test a service in isolation, we use docker compose up, if we want the full end to end service we use the start script.  I see the test compose file uses the isolated message queue which makes sense.